docs/document/content/reference/shadow/_index.en.md

```diff
@@ -248,4 +248,29 @@ props:
   sql-comment-parse-enabled: true
 ```
 
-**Note**: The default shadow algorithm only supports note shadow algorithm.
+**Note**:
+The default shadow algorithm only supports note shadow algorithm.
+When using HINT, ensure that the configuration items of `props` in the configuration file are less than or equal to those in the SQL comment, and that the configuration items in the configuration file are the same as those in the SQL comment. The fewer the configuration items in the configuration file, the looser the matching conditions are
```

Comment:
I agree with what you said, but the purpose of adding this document is to make others better understand the matching mode of rules, because the configuration of this part is less described in the current document. At the same time, I think my change does not induce others to use it, but to tell others how rules are matched